controllers/user.js

```diff
@@ -337,6 +338,20 @@ async function getPreferences(req, res) {
 }
 
 
+async function getBreachStats(req, res) {
+  const fxaResponse = await FXA.verifyOAuthToken(req.query.fxa_oauth_token);
```

Comment:
I'm guessing if an `fxa_auth_token` wasn't passed via the querystring, that'd just be `null` or `undefined`. Not sure if we'd need to do a check for a valid-looking oauth token before proceeding or if that is just overkill.

Reply:
Added checks and appropriate responses.